pkg/pools/pools.go

```diff
@@ -135,3 +149,10 @@ func (bufPool *BufioWriterPool) NewWriteCloserWrapper(buf *bufio.Writer, w io.Wr
 		return nil
 	})
 }
+
+func subsequentCloseWarn(name string) {
+	log.G(context.TODO()).Warn("subsequent attempt to close " + name)
```

Comment:
```suggestion
	log.G(context.TODO()).Error("BUG: subsequent attempt to close " + name)
```
Warnings are for users, IMO. Buggy program behaviour is an error.